mne/preprocessing/infomax_.py

```diff
@@ -199,9 +208,8 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
             # ICA kurtosis estimation
             if extended is True:
 
-                n = np.fix(blockno / ext_blocks)
+                if (ext_blocks > 0) and ((blockno % ext_blocks) == 0):
```

Comment:
parentheses not needed.
we make an exception for assignment statments where the assignment operator can be confused with the logical comparison